src/stage/stage.ts

```diff
@@ -102,6 +108,8 @@ declare global {
  * @property {Color} ambientColor - ambient light color
  * @property {Float} ambientIntensity - ambient light intensity
  * @property {Integer} hoverTimeout - timeout for hovering
+ * @property {Float} expandBoundingBox - expand bounding box on all dimensions by this factor
```

Comment:
I think `expandBoundingBoxFactor` would be a clearer name. Just  `expandBoundingBox` sounds to me like adding that many units to it.

Reply:
I'll make that change. 
I didn't intend to push this commit into this PR, actually (thanks github), but now that it's here, let me explain the reason for it. I'm adding some Three.js objects into the scene inside `scene/rotationGroup/translationGroup`. They're bigger than the molecule. Without a change like this, they get fogged and clipped, because the bbox is set to the molecule size. (The Three.js objects are just straight-up Three.js geometry, not using the NGL object system.)
So if there's an alternate way to have the bbox include arbitrary other Three.js objects in the scene, that would be better than my change here. Any ideas appreciated!